tests/unit/BitbucketContext/Application/EventListener/PullRequestBuildStatusListenerTest.php

```diff
@@ -130,6 +130,22 @@ public function testItDoesNothingIfTheRepositoryDisabledTheInspections()
         $this->listener->onInspectionStarted($event);
     }
 
+    public function testItDoesNothingIfTheRepositoryIsInFlightMode()
+    {
+        $inspection = $this->createMock(Entity\PullRequestInspection::class);
+        $repository = $this->createMock(Entity\Repository::class);
+        $domainEvent = new Event\InspectionStarted($inspection);
```

Comment:
Parameter #1 $inspection of class Regis\BitbucketContext\Application\Event\InspectionStarted constructor expects Regis\BitbucketContext\Domain\Entity\PullRequestInspection, PHPUnit_Framework_MockObject_MockObject given.